portfolio.html

```diff
@@ -20,46 +20,279 @@
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default">See Project</button>
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/yoga.png" alt="yoga" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Tonic</h1>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="seprator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="seprator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default">See Project</button>
+              </div>
+            </div>
+          </article>
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/art.png" alt="yoga" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="seprator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="seprator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="about bg-white">
+          <h1 class="title">About Myself</h1>
```

Comment:
- [ ] Kindly use headings properly in your codes to make your work more professional. look more on [headings](https://www.w3schools.com/html/html_headings.asp)